adapter/internal/discovery/xds/marshaller.go

```diff
@@ -78,6 +78,15 @@ func MarshalConfig(config *config.Config) *enforcer.Config {
 		ExpiryTime:  config.Enforcer.Cache.ExpiryTime,
 	}
 
+	tracing := &enforcer.Tracing{
+		Enabled:		 		config.Enforcer.Tracing.Enabled,
+		Type:					config.Enforcer.Tracing.Type,
+		ConfigProperties:		config.Enforcer.Tracing.ConfigProperties,
```

Comment:
formatting issue

Reply:
Fixed in 8d52822bdbad80f19fc2738c59f60af36e75e5c8